### ISSUE TITLE
Attempted to measure layout but offset or dimensions were NaN

### ISSUE THREAD

**Issue by lazywei** [facebook/react-native#1793]
I got this error, but I don't think I have used layout measure in somewhere. Is there any other more useful way to check what is the cause?

I'm using react-native 0.7.0-rc

```
2015-06-30 12:26:31.016 [error][tid:com.facebook.React.ShadowQueue][RCTUIManager.m:1062] Attempted to measure layout but offset or dimensions were NaN
2015-06-30 12:26:31.019 [error][tid:com.facebook.React.ShadowQueue][RCTUIManager.m:1062] Attempted to measure layout but offset or dimensions were NaN
```

Thanks

**Comment by ide** [comment]
Update to 0.7.0-rc.2, which fixes this in ListViews.

**Comment by lazywei** [comment]
I confirmed this issue is fixed in 0.7.0-rc.2. Thanks @ide
